Doc/library/enum.rst

```diff
@@ -761,11 +761,11 @@ Data Types
 Supported ``__dunder__`` names
 """"""""""""""""""""""""""""""
 
-:attr:`__members__` is a read-only ordered mapping of ``member_name``:``member``
+:attr:`!__members__` is a read-only ordered mapping of ``member_name``:``member``
 items.  It is only available on the class.
 
-:meth:`__new__`, if specified, must create and return the enum members; it is
-also a very good idea to set the member's :attr:`_value_` appropriately.  Once
+:meth:`!__new__`, if specified, must create and return the enum members; it is
```

Comment:
Unless there's a clear reason not to, why not just make this a proper xref to the canonical documentation of the `__new__` dunder, so users unfamiliar with it can have the appropriate background?

```suggestion
:meth:`~object.__new__`, if specified, must create and return the enum members; it is
```